lib/util/fast_jason_encoder.ex

```diff
@@ -0,0 +1,68 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoder do
+  @moduledoc """
+  This module converts structures to JSON at low load by avoiding protocol calls. Internally jason is used.
+  """
+  alias Antikythera.Time
+
+  defmodule Wrapper do
+    defstruct item: nil
+  end
+
+  defimpl Jason.Encoder, for: Wrapper do
+    def encode(%Wrapper{item: item}, opts) do
+      Antikythera.FastJasonEncoder.encode(item, opts)
+    end
+  end
+
+  def encode(value) do
```

Comment:
(optional)
I think you should use `Croma.Defun.defun` to clarify the type in public functions that might be widely used.
(The following example requires `alias Croma.Result, as: R`.)
```suggestion
  defun encode(value :: any) :: R.t(String.t, Jason.EncodeError.t | Exception.t) do
```

Reply:
Fixed e3ab6a3e6831b012938f7317a79e3f500de1ab8c